python/requirements.txt

```diff
@@ -1,4 +1,5 @@
 Cython==3.0a6
+decorator
```

Comment:
why we need `decorator` ?

Reply:
Used in decorator `not_compatible_for`

Reply:
`not_compatible_for` seems it does not need `decorator`?